### PR TITLE
issue #170

### PR DESCRIPTION
**Pull request by ishankhare07**
added the start command

### CODE REVIEW

File: rbkit-lib/subscriber.cpp [hunk 1]
```diff
@@ -229,7 +235,7 @@ void Subscriber::performHandshake()
     try
     {
         commandSocket->start(commandUrl);
-        switch (commandSocket->performHandShake()) {
+        /*switch (commandSocket->performHandShake()) {
```
Comment (iffyuva) [inline rbkit-lib/subscriber.cpp]: can you please uncomment this part?

**Comment by iffyuva** [comment]
@ishankhare07 please don't put `#` in branch name.

File: rbkit-lib/subscriber.cpp [hunk 2]
```diff
@@ -55,6 +55,12 @@ void Subscriber::triggerGc() {
     commandSocket->sendCommand(triggerGC_Command);
 }
 
+void Subscriber::startCPUProfiling() {
+    RBKit::CmdStartProfile startCPUProfile;
```
Comment (iffyuva) [inline rbkit-lib/subscriber.cpp]: the command used here is wrong. it sends `start_memory_profile` command to backend.

Comment (iffyuva) [inline rbkit-lib/subscriber.cpp]: @emilsoman what is the command which needs to be sent to backend? `start_cpu_profile` ?

Comment (emilsoman) [inline rbkit-lib/subscriber.cpp]: `start_cpu_profiling` and `stop_cpu_profiling` . We also need another command to set the type of clock to use (wall/cpu).

File: rbkit-lib/subscriber.cpp (hunk 1 above)
Comment (ishankhare07) [inline rbkit-lib/subscriber.cpp]: uncommented

File: rbkit-lib/subscriber.cpp (hunk 2 above)
Comment (ishankhare07) [inline rbkit-lib/subscriber.cpp]: added `start_cpu_profiling`
also added `stop_cpu_profiling` in branch #172 

had to add new classes in ```rbcommands.h``` and ```rbcommands.cpp```

**Pull request merged**